lib/CL/pocl_cl.h

```diff
@@ -794,6 +794,8 @@ struct _cl_device_id {
   cl_uint dev_queue_max_size;
   cl_command_queue_properties on_dev_queue_props;
   cl_command_queue_properties on_host_queue_props;
+  /* OpenCL 2.1 */
```

Comment:
What does this comment want to say? That SPIR-V is a requirement since 2.1, I think?

Reply:
it's "OpenCL 2.1 related properties follow", same as OpenCL 2.0 comment a few lines above